src/components/user/PersonalProfileDetails.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from "react";
+import {
+  makeStyles,
+  Container,
+  Typography,
+  Button,
+  Grid,
+  Box
+} from "@material-ui/core";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import User from "../../service/models/user.model";
+import ChangePasswordModal from "./ChangePasswordModal";
+import UpdateProfileDetailsForm from "./UpdateProfileDetailsForm";
+
+type Props = {
+  user: User;
+};
+
+const GAP_BETWEEN_INPUTS = 10;
+
+const useStyles = makeStyles(theme => ({
+  personalProfileDetailsLayout__mainContainer: {
+    marginTop: 20,
+    marginBottom: 20,
+    padding: "0 40px"
+  },
+  personalProfileDetailsLayout__subText: {
+    textAlign: "left",
+    color: "#cccccc",
+    marginBottom: GAP_BETWEEN_INPUTS
+  },
+  personalProfileDetailsLayout__submitBtn: {
+    display: "block",
+    marginTop: 40,
+    marginBottom: 40
+  },
+  personalProfileDetailsLayout__modal: {
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center"
+  },
+  personalProfileDetailsLayout__paper: {
+    backgroundColor: theme.palette.background.paper,
+    border: "2px solid #000",
+    boxShadow: theme.shadows[5],
+    padding: theme.spacing(2, 4, 3)
+  }
+}));
+
+function PersonalProfileDetails(props: Props) {
+  const { user } = props;
+  const styles = useStyles();
+  const [open, setOpen] = useState(false);
+  //const [error, setError] = useState("");
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const [inputData, setInputData] = useState({
+    password: "",
+    confirmPassword: "",
+    lastName: user.lastName,
+    firstName: user.firstName,
+    civility: "",
+    functionTitle: user.functionTitle,
+    emailDisplay: user.email,
+    phoneNumber: "",
+    whatsAppNumber: "",
+    weChatId: "",
+    skypeId: ""
```

Comment:
We need to check with Diane for that - to me one contact field seems enough, in addition of phone & email. We can then define an enum and add a dropdown list

Reply:
Ok